pyproject.toml

```diff
@@ -51,8 +52,12 @@ sqlalchemy_utils = "^0.41.1"
 strawberry-graphql = "^0.217.0"
 
 [build-system]
-requires = ["poetry-core"]
-build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core", "poetry-dynamic-versioning>=1.0.0,<2.0.0"]
```

Comment:
How does poetry-dynamic-versioning figure out what version to use? Does it look at a git tag or something?